src/frontends/pytorch/src/transforms/tuple_unpack_replacer.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "tuple_unpack_replacer.hpp"
+
+#include "openvino/op/util/framework_node.hpp"
+#include "openvino/pass/pattern/matcher.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace pass {
+
+PrimTupleUnpackReplacer::PrimTupleUnpackReplacer() {
+    auto tuple_unpack = ov::pass::pattern::wrap_type<ov::op::util::FrameworkNode>();
+
+    ov::matcher_pass_callback callback = [](ov::pass::pattern::Matcher& m) {
+        auto tuple_unpack = cast_fw_node(m.get_match_root(), "prim::TupleUnpack");
+        if (!tuple_unpack)
+            return false;
+        OutputVector outputs;
+        NodeVector to_copy_rt;
+        auto input_node = tuple_unpack->get_input_node_shared_ptr(0);
+        auto tuple_construct = cast_fw_node(input_node, "prim::TupleConstruct");
+        if (!tuple_construct) {
+            return false;
+        }
+        for (const auto& input : input_node->inputs()) {
+            const auto& out = input.get_source_output();
+            outputs.push_back(out);
+            to_copy_rt.push_back(out.get_node_shared_ptr());
+        }
+        copy_runtime_info({tuple_unpack, input_node}, to_copy_rt);
```

Comment:
Not sure that this is a correct RT transfer. `to_copy_rt` contains all nodes that provides inputs to `TupleConstruct`, why do we need to fuse RT into all of them? Looks like there is no any other place suitable for that but can we just lose this info? What are the implications?

Reply:
I think there is no more impack here if we will no copy info as there is no additional context between packing and unpacking of these nodes, so I removed copying